onboarding/source/amb_light_service.c

```diff
@@ -6,7 +6,19 @@
 
 /* USER CODE BEGIN */
 // Include any additional headers and global variables here
+#include "FreeRTOS.h"
+#include "os_task.h"
+#include "os_queue.h"
 
+#include <string.h>
+#include <stdio.h>
+
+void adcStartConversion_selChn(adcBASE_t *adc, uint8_t channel, uint8_t fifo_size, uint8_t group);
+void adcGetSingleData(adcBASE_t *adc, unsigned group, adcData_t *data);
+uint16_t getADCConversion();
```

Comment:
In C, when a function takes no arguments, you should write `void` as the argument. For example, something like `uint16_t getADCConversion(void);`

Reply:
Done